### PR TITLE
Add rating metric in Terraforming Mars

### PR DESCRIPTION
**Pull request by tonycdot**

### CODE REVIEW

**Comment by flungloaf** [comment]
What is this?

**Comment by maksimf** [comment]
9602e410f5cda88aaa033d2658e2386a

**Comment by tonycdot** [comment]
Terraforming rating, first metric

File: src/pages/games/terraforming-mars/Match.tsx [hunk 1]
```diff
@@ -22,7 +22,7 @@ import { useRouteMatch } from "react-router-dom";
 import theme from "../../../theme";
 import { Rating } from "@material-ui/lab";
 
-const metrics = ["achievements", "awards", "cards", "cities", "forests"];
+const metrics = ["rating", "achievements", "awards", "cards", "cities", "forests"];
```
Comment (flungloaf) [inline src/pages/games/terraforming-mars/Match.tsx:25]:
```suggestion
const metrics = ["terraform rating", "achievements", "awards", "cards", "cities", "forests"];
```
This also is more than default line length for prettier, run `prettier:fix`

**Review by flungloaf — CHANGES_REQUESTED** [review]

**Review by flungloaf — APPROVED** [review]

**Pull request merged**